pkg/collector/corechecks/net/ntp.go

```diff
@@ -126,6 +146,28 @@ func (c *ntpConfig) parse(data []byte, initData []byte, getLocalServers func() (
 	return nil
 }
 
+func getCloudProviderNTPHosts() []string {
+	if ec2.IsRunningOn() || ecs.IsRunningOn() {
```

Comment:
Let create a new method `GetNTPHosts` instead with tests for each of them.

Also let log the actual hosts used so the customers can see them (maybe as `debug` ?).

Reply:
I'm not sure I understand. What would be the difference to this method? Do you mean one `GetNTPHosts` for each cloud provider package?